test/models/database/vote/vote_firestore_test.dart

```diff
@@ -0,0 +1,20 @@
+import "package:flutter_test/flutter_test.dart";
+import "package:proxima/models/database/vote/vote_firestore.dart";
+
+void main() {
+  group("Vote Firestore testing", () {
+    test("hash overrides correctly", () {
+      final voteFirestore = VoteFirestore(hasUpvoted: true);
+      final hashCode = voteFirestore.hashCode;
+
+      expect(hashCode, isA<int>());
```

Comment:
Why are you just testing that `hashCode` is an `int` and not that the expected hashcode is the same as the one computed from `voteFirestore`

Reply:
You are completely right, this is a mistake.